src/pages/About.js

```diff
@@ -1,20 +1,16 @@
 import React, { Component } from 'react';
-class About extends Component {
 
-    render () {
-        return (
-            <div>
-                <h1>About BoilerMake</h1>
-                <a href="http://www.nicky.photos/Events/BoilerMake/i-dntZB2F/A"><img width="80%" src="https://photos.smugmug.com/Events/BoilerMake/i-dntZB2F/0/XL/DSC_4887-2-XL.jpg" alt=""/></a>
-                <br/>
-                <br/>
-                <a href="http://www.nicky.photos/Events/BoilerMake-4/i-dGvGsSB/A"><img width="80%" src="https://photos.smugmug.com/Events/BoilerMake-4/i-dGvGsSB/0/X2/NDS_2936-2-XL.jpg" alt=""/></a>
-                <br/>
-                <p>Boilermake is an organization run by students at Purdue University that brings 500 students from all over to Purdue to compete in a 36 hour hackathon. We partner and we work with sponsors to create the optimal hacking space and recruiting space for the attendees. Our vision is to support the hacking culture to help create an innovative future.</p>
+const About = () => (
```

Comment:
heh, i think i wrote this before i knew about stateless components ty for the cleaning 👍